src/_variables.tsx

```diff
@@ -0,0 +1,7 @@
+export const variables = {
```

Comment:
do we need this?

Reply:
I have two components where I am using it as I cannot use variables.scss inside tsx files.
I use it in MyButton.tsx and in HomePage.tsx